Schema/authn/002-fix-login.blue.sql

```diff
@@ -27,7 +27,7 @@ CREATE FUNCTION odin.credentials_ledger_insert() RETURNS TRIGGER AS $body$
             INTO odin.credentials
                 (identity_id, login, password__hash, password__process, password__reference)
             VALUES (NEW.identity_id, NEW.login, NEW.password, NEW.process, NEW.reference)
-            ON CONFLICT (login) DO UPDATE SET
+            ON CONFLICT (identity_id) DO UPDATE SET
```

Comment:
Is there already a uniqueness constraint on the login name?

Reply:
Yes it is. https://github.com/KayEss/odin/blob/feature/fix-register-view/Schema/authn/002-fix-login.blue.sql#L10